public/js/apps/payment/main.js

```diff
@@ -1,8 +1,6 @@
-'use strict';
-
-var $ = require('jquery');
-var tracking = require('tracking');
-var App = require('./app');
+import $ from 'jquery';
+import tracking from 'tracking';
```

Comment:
I don't see an export named `tracking` -- does it work because that's the module name? Is it shorthand for `import { default as tracking } from 'tracking'` ?